src/style.css

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com");
```

Comment:
The URL for the 'Roboto' font is incorrect. Use the correct URL from Google Fonts: https://fonts.googleapis.com/css?family=Roboto